guide/like.md

```diff
@@ -0,0 +1,10 @@
+## User Handbook
+### Liking a post
```

Comment:
I can add a note at the top explaining that, do you have any other functions in mind that would need this also? as far as I know everything (this included) is enabled by default, isn't it?